src/ralph/models/xapi/video/constants.py

```diff
@@ -5,12 +5,47 @@
 # xAPI video extensions
 VIDEO_EXTENSION_SESSION_ID = "https://w3id.org/xapi/video/extensions/session-id"
 VIDEO_EXTENSION_TIME = "https://w3id.org/xapi/video/extensions/time"
-
+VIDEO_EXTENSION_COMPLETION_THRESHOLD = (
+    "https://w3id.org/xapi/video/extensions/completion-threshold"
+)
+VIDEO_EXTENSION_LENGTH = "https://w3id.org/xapi/video/extensions/length"
+VIDEO_EXTENSION_PLAYED_SEGMENTS = (
+    "https://w3id.org/xapi/video/extensions/played-segments"
+)
+VIDEO_EXTENSION_PROGRESS = "https://w3id.org/xapi/video/extensions/progress"
+VIDEO_EXTENSION_CC_SUBTITLE_LANG = (
+    "https://w3id.org/xapi/video/extensions/cc-subtitle-lang"
+)
+VIDEO_EXTENSION_CC_SUBTITLE_ENABLED = (
+    "https://w3id.org/xapi/video/extensions/cc-enabled"
+)
+VIDEO_EXTENSION_FRAME_RATE = "https://w3id.org/xapi/video/extensions/frame-rate"
+VIDEO_EXTENSION_FULL_SCREEN = "https://w3id.org/xapi/video/extensions/full-screen"
+VIDEO_EXTENSION_QUALITY = "https://w3id.org/xapi/video/extensions/quality"
+VIDEO_EXTENSION_SCREEN_SIZE = "https://w3id.org/xapi/video/extensions/screen-size"
+VIDEO_EXTENSION_VIDEO_PLAYBACK_SIZE = (
+    "https://w3id.org/xapi/video/extensions/video-playback-size"
+)
+VIDEO_EXTENSION_SPEED = "https://w3id.org/xapi/video/extensions/speed"
+VIDEO_EXTENSION_TRACK = "https://w3id.org/xapi/video/extensions/track"
```

Comment:
Not defined in xapi

Reply:
Do you search for this extension  https://liveaspankaj.gitbooks.io/xapi-video-profile/content/statement_data_model.html#26210-track ?

Reply:
That's it, yes! Thanks! Should not have written it openly, it was notes in my WIP, sorry...